physionet-django/user/forms.py

```diff
@@ -585,7 +586,7 @@ def clean(self):
                        data['reference_organization'],
                        data['reference_title']]
 
-        ref_required = data['researcher_category'] in [0, 1, 6, 7]
+        ref_required = data['researcher_category'] in range(8)
```

Comment:
This looks like it adds a requirement for a reference for all categories? I think the only change we wanted to make was to require a reference for independent researchers?

Reply:
See: https://github.com/MIT-LCP/physionet-build/blob/dev/physionet-django/user/models.py#L707-L720 for the list of categories:

```python
class CredentialApplication(models.Model):
    """
    An application to become credentialed
    """
    RESEARCHER_CATEGORIES = (
        (0, 'Student'),
        (7, 'Graduate Student'),
        (1, 'Postdoc'),
        (2, 'Academic Researcher'),
        (3, 'Hospital Researcher'),
        (4, 'Industry Researcher'),
        (5, 'Government Researcher'),
        (6, 'Independent Researcher'),
    )
```